fastlane/Fastfile

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+default_platform(:ios)
+
+platform :ios do
+  desc 'Builds the project and runs tests'
+  lane :test do
+    run_tests(
+      project: 'MediaEditor.xcodeproj',
+      scheme: 'MediaEditor',
+      devices: ['iPhone 11'],
+      deployment_target_version: '14.5',
```

Comment:
Have you tried the latest for Xcode 13.0 (used in CI), which I guess would be 15?

Reply:
FWIW, I tried this on my machine and it worked

```suggestion
      deployment_target_version: '15.0',
```

Reply:
So...as identified in https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1349/files#diff-dff4b99d4e651ab9d7597876ec8dbe92aa934e42c0fb55f4aadd6c106fc96688R37, this value really doesn't do much (it'll use iOS 15 under the hood no matter what this is set to).

I figure we can run around and update all our projects once:
1. That PR lands
2. I'm able to extract the simulator creation bits into the release toolkit, then update all our projects to validate the available simulator versions.